CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing
+
+Implementing and maintaining a protocol stack is a lot of work, therefore any
+help is appreciated, from creating issues, to contributing documentation, fixing
+issues and adding new features.
+
+Please follow the best-practice contribution guidelines as mentioned below when
+submitting any changes.
+
+### Contributions License Agreement
+
+Before contributing, please read the [Contributions License Agreement](./CLA.md).
+Every contributor must sign this agreement before any code is merged into the
+repository.
+
+### Git
+
+Please, do your changes with the develop branch in your forked repo
+and send us your pull request from your own forked repo feature-branches
+or fix-branches to the develop branch of our repo!
+
+In the case that Commit rights were granted to Contributors the following basic ground-rules should be applied:
+
+* No --force pushes or modifying the Git history in any way.
+* Non-master branches ought to be used for ongoing work.
```

Comment:
What do you mean by this?
I am not I follow.
Maybe an example would be good to make it clear for everyone?

Reply:
I remove it

Reply:
I adjusted it